source/uwp/Renderer/dll/dll.cpp

```diff
@@ -5,12 +5,13 @@
 #include <windows.h>
 #include <wrl.h>
 #include <wrl\wrappers\corewrappers.h>
+#include <module.g.cpp>
 
 using namespace Microsoft::WRL;
 
 STDAPI DllGetActivationFactory(_In_ HSTRING activatableClassId, _COM_Outptr_ IActivationFactory** factory)
 {
-    return Module<ModuleType::InProc>::GetModule().GetActivationFactory(activatableClassId, factory);
+    return WINRT_GetActivationFactory(activatableClassId, reinterpret_cast<void**>(factory));
 }
 
 STDAPI DllGetClassObject(_In_ REFCLSID rclsid, _In_ REFIID riid, _COM_Outptr_ void** ppv)
```

Comment:
>DllGetClassObject [](http://example.com/codeflow?start=7&length=17)

I'm wondering if still need this

Reply:
@paulcam206 why do we need it though?:)
is that if somebody wants to use our nuget package with **WRL**?

Reply:
@oldnewthing can you please provide some guidance here?